.pre-commit-config.yaml

```diff
@@ -0,0 +1,45 @@
+---
+repos:
+  # Formats import order
+  - repo: https://github.com/pycqa/isort
+    rev: 5.12.0
+    hooks:
+      - id: isort
+        name: isort (python)
+        args: ["--profile", "black", "--filter-files"]
+
+  # Code formatter for both python files and jupyter notebooks
+  - repo: https://github.com/psf/black
+    rev: 22.10.0
+    hooks:
+      - id: black-jupyter
```

Comment:
Seeing how the repo reference below might influence things, would it make sense to rely on `nbqa-black` from the `nbQA` project (instead of `black-jupyter`)?